fiona/ogrext1.pyx

```diff
@@ -151,6 +153,8 @@ cdef class FeatureBuilder:
         cdef int mm = 0
         cdef int ss = 0
         cdef int tz = 0
+        cdef unsigned char *data
+        cdef int l
```

Comment:
This doesn't work. OGR_F_GetFieldAsBinary needs a pointer to an int so that it can write the output to it. I think it's correct as it is.